client/checkout/platform-checkout/express-button/woopay-express-checkout-button.js

```diff
@@ -2,19 +2,39 @@
  * External dependencies
  */
 import { sprintf, __ } from '@wordpress/i18n';
+import { useEffect } from 'react';
 
 /**
  * Internal dependencies
  */
 import WoopayIcon from './woopay-icon';
 import { expressCheckoutIframe } from './express-checkout-iframe';
+import wcpayTracks from 'tracks';
 
 export const WoopayExpressCheckoutButton = ( {
 	isPreview = false,
 	buttonSettings,
 	api,
 } ) => {
-	const { type: buttonType, text, height, size, theme } = buttonSettings;
+	const {
+		type: buttonType,
+		text,
+		height,
+		size,
+		theme,
+		context,
+	} = buttonSettings;
+
+	useEffect( () => {
+		if ( ! isPreview ) {
+			wcpayTracks.recordUserEvent(
+				'platform_checkout_express_button_offered',
```

Comment:
I'm not super familiar with these track events, would this "offered" event cause too much noise? I noticed the `PLATFORM_CHECKOUT_OFFERED` is only recorded after a user exists check, while this one would be recorded I believe on every page load of a product, cart, or checkout (settings dependent of course). This may be a great thing to track, but it was just a thought I had when I saw it.

Reply:
Yeah, I agree that there could be a lot of noise, but I think we would need to track it so we can see what the conversion rate is (button clicks/button offered).